python/ray/tests/test_task_events_2.py

```diff
@@ -1118,90 +1118,13 @@ def verify_tasks(expected_tasks_cnt: Dict[str, int]):
     reason="setproctitle has different definitions of `title` on different OSes",
 )
 class TestIsActorTaskRunning:
-    def test_main_thread_short_comm(self, ray_start_regular):
```

Comment:
See https://github.com/ray-project/ray/pull/51158 for more details. The Ray task is no longer running on the main thread, so these tests are not needed.